README.md

```diff
@@ -9,47 +9,66 @@
 
-The location log generated by Safe Paths cannot be accessed from outside the user’s device. However, the user can import and export their location information and use it in other projects and applications.
+## Multiple Implementation Strategies 
 
-Safe Paths logs your device’s location once every five minutes and stores 14 days of data.
+Safe Paths is designed to support a variety of implementation strategies. 
 
-### Private Kit WhitePaper
+### Path Check Release of COVID Safe Paths
+Safe Paths is available as an app published by Path Check in the [Apple App Store](https://apps.apple.com/us/app/covid-safe-paths/id1508266966) and the [Google Play App Store](https://play.google.com/store/apps/details?id=org.pathcheck.covidsafepaths). Any authorized pubic health authority can use Safe Paths. 
 
-[Apps Gone Rogue: Maintaining Personal Privacy in an Epidemic](https://drive.google.com/file/d/1nwOR4drE3YdkCkyy_HBd6giQPPhLEkRc/view?usp=sharing)
+### Custom Builds 
 
-### Downloads for COVID Safe Paths
+We welcome public health authorities and other organizations implementing digital contact tracing strategies to create custom builds for their specific needs, incorporate Safe Paths features into their applications, or create downstream projects that stay linked to the Safe Paths project. 
 
-[Google Play](https://play.google.com/store/apps/details?id=org.pathcheck.covidsafepaths) | [Apple Store](https://apps.apple.com/us/app/covid-safe-paths/id1508266966)
+### GAEN Instances
+
+The rules for GAEN require that each public health authority release their own application. The goal with Project Aurora is to suppor this deployment strategy. 
+
+## End-to-End System
+
+Safe Paths is designed to work with Safe Places, a tool for contact tracing teams to work with location data in the course of contact tracing interviews and to publish points of concern. Without having to sharing their own location history, a Safe Paths user can download the points of concern from their PHA to identify if they have had risk of an exposure. 
+
+Project Aurora supports any GAEN compliant Key Server for exposure notification. 
 
-### Downloads for Private Kit (technology beta)
+## Broad Non-Developer Community 
 
-[Google Play](https://play.google.com/store/apps/details?id=edu.mit.privatekit) | [Apple Store](https://apps.apple.com/us/app/private-kit-prototype/id1501903733)
+One of the important aspects of the Safe Paths open source project is that it's supported by a large community of volutneers in addition to the open source developer community. More than 1,400 Path Check volunteers are working together to support Safe Paths and help drive adoption around the world. 
```

Comment:
One of the important aspects of the Safe Paths open source project is that it's supported by a large community of volunteers in addition to the open source developer community. Spanning as diverse domains as product management, user research, cryptography, security, compliance, design, and videography more than 1,400 Path Check volunteers are working together to support Safe Paths and help drive adoption around the world.